library/src/test/java/bio/terra/pfb/PfbReaderTest.java

```diff
@@ -0,0 +1,93 @@
+package bio.terra.pfb;
+
+import static bio.terra.pfb.utils.CompareOutputUtils.FileExtension.JSON;
+import static bio.terra.pfb.utils.CompareOutputUtils.FileExtension.TXT;
+import static bio.terra.pfb.utils.CompareOutputUtils.PfbCommandType.*;
+
+import bio.terra.pfb.utils.CompareOutputUtils;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class PfbReaderTest {
+  private static final Logger logger = LoggerFactory.getLogger(PfbReaderTest.class);
+  private static final List<String> listOfTestFiles =
+      List.of("minimal_schema", "test", "kf", "data-with-array", "empty");
+
+  @Test
+  void showSchemaTest() throws IOException {
+    // TODO - remove when fix is added for AJ-1288
+    var editedListOfTestFiles = listOfTestFiles.stream().filter(f -> !f.equals("empty")).toList();
+    for (String fileName : editedListOfTestFiles) {
+      CompareOutputUtils.compareJavaPfbWithPyPfb(fileName, SHOW_SCHEMA, "", JSON);
+    }
+  }
+
+  @Test
+  void showNodesTest() throws IOException {
```

Comment:
Sonar is going to flag this because it doesn't contain any asserts. If you rename `compareJavaPfbWithPyPfb()` to a name with `assert` like `assertJavaPfbIsPyPfb()` then the sonar issue should go away.